### PR TITLE
Table header edits & tooltips added

### PR DESCRIPTION
**Pull request by JulianForeman**
# Test Links:
[Percentile Calculator](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1484.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/fbaCalculator/components/FBATableHead.tsx [hunk 1]
```diff
@@ -176,23 +175,20 @@ const FBATableHead = ({
...
       <TableCell className={classes.windSpeed} sortDirection={order}>
         <TableSortLabel
+          className={classes.windSpeed}
           direction={order}
           onClick={() => {
             toggleSorting(SortByColumn.WindSpeed)
           }}
         >
-          Wind Speed (km/h)
+          <TableHeader text={'Wind Speek (km/h)'}></TableHeader>
```
Comment (conbrad) [inline web/src/features/fbaCalculator/components/FBATableHead.tsx:191]: `Speek` -> `Speed`

**Review by tessmgood — CHANGES_REQUESTED** [review]
Looking great! Last small change, something wonky is happening in this tooltip, the end parenthesis is trying to escape the container
Screen Shot 2021-11-01 at 2 09 26 PM

**Review by tessmgood — APPROVED** [review]
thanks!

File: web/src/features/fbaCalculator/components/TableHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        position: 'absolute',
+        left: '2px',
+        top: '41px'
+      }
+    }
+  })
+
+  const classes = useStyles()
+  return (
+    <div style={{ maxWidth: props.maxWidth }} className={classes.header}>
```
Comment (conbrad) [inline web/src/features/fbaCalculator/components/TableHeader.tsx:39]: Please put this in the class above so we're consistent with how we add CSS (no inline styles).

You can access props inside the component like `FireTable` does: https://github.com/bcgov/wps/blob/c0bcc35ed15e4616885a5caa89d21ccd5504fd6a/web/src/components/FireTable.tsx#L23

**Review by conbrad — CHANGES_REQUESTED** [review]
Just one more change, looks good!

**Pull request merged**